drivers/hwinfo/Kconfig

```diff
@@ -21,55 +24,63 @@ config HWINFO_STM32
 	default y
 	depends on SOC_FAMILY_STM32
 	select USE_STM32_LL_UTILS
```

Comment:
This could actually be removed since driver only require headers, not utils functions

Reply:
Thanks. Removed.